clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -81,6 +81,7 @@ class AutoScalingWorker {
   private int desiredInstances
 
   private RegionScopedProviderFactory.RegionScopedProvider regionScopedProvider
+  private Integer maxSecurityGroupsPerInstance
```

Comment:
`maxSecurityGroups` vs `maxSecurityGroupsPerInstance` as the config property name?